Equinor.Maintenance.API.EventEnhancer/Program.cs

```diff
@@ -132,7 +132,7 @@
             {
                 var result = await mediator.Send(new PublishMaintenanceEventQuery(body), cancelToken);
 
-                return result.StatusCode < 399 ? Results.Created(string.Empty, result.Data) : Results.StatusCode(result.StatusCode);
+                return result.StatusCode < 399 ? Results.Created(new Uri("www.equinor.com"), result.Data) : Results.StatusCode(result.StatusCode);

```

Comment:
In the future we will probably just return a simple 200, 202 or 204 since the event source only cares a about a successful response code.

Reply:
Yes we shall :ez: